js-exercises/negative-index/negativeIndex.test.js

```diff
@@ -0,0 +1,39 @@
+import { negativeIndex } from './negativeIndex';
+
+describe('negativeIndex', () => {
+  test('behaves like an array', () => {
+    const fixture = negativeIndex(['foo', 'bar', 'baz']);
+    expect(fixture.length).toBe(3);
+    expect(fixture.toString()).toBe('foo,bar,baz');
+  });
+
+  test('get values', () => {
+    const fixture = negativeIndex(['foo', 'bar', 'baz']);
+    expect(fixture[0]).toBe('foo');
+    expect(fixture[1]).toBe('bar');
+    expect(fixture[-1]).toBe('baz');
+    expect(fixture[-2]).toBe('bar');
+  });
+
+  test('set values', () => {
```

Comment:
Test case is failing for this scenario.Please consider handling this.

Reply:
Yes, I will fix this! I am stuck at this point!